### DISCUSSION TITLE
Somehow Android target API was changed in build time

### DISCUSSION THREAD

**Discussion by pkar70** [unoplatform/uno#15913] [Q&A]
### Current behavior

I ported one of my apps to Uno, and I cannot publish it to Google Store - it complains about target API.

Settings in Vstudio:
![image](https://github.com/unoplatform/uno/assets/23451507/37219bea-a09e-4e24-ad29-b2c5ec5cdeb2)

Complain from Google:
![image](https://github.com/unoplatform/uno/assets/23451507/5f5b125d-4998-49a8-a9f1-bfc451d79320)

But Google mean Target, not minimum:
«Configuring your app to target a recent API level ensures that users benefit from security, privacy, and performance improvements, while still allowing an app to run on older Android versions (down to the specified minSdkVersion).»

When I change minimum target to 30, Google still complains about target 28.

### Expected behavior

No change in target level.

### How to reproduce it (as minimally and precisely as possible)

https://github.com/pkar70/EduQuiz/tree/master/QuizKursUno

### Works on UWP/WinUI

None

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

5.1.52

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

17.9.0

**Comment by pkar70** [comment]
When I checked AndroidManifest.xml files in obj\debug/release (over 100 files), most frequenty occurs line
`<uses-sdk android:minSdkVersion="14" />` , and in `\obj\Debug\net8.0-android34.0\android\manifest\AndroidManifest.xml` line is `  <uses-sdk android:minSdkVersion="28" android:targetSdkVersion="28" />`

**Reply by jeromelaban** [reply]
@pkar70 this is likely a behavior coming from .NET, which could be a change coming from the msbuild `TargetPlatformVersion` property. You can troubleshoot this using binlogs.
